recipes/pygsl_lite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pygsl_lite" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygsl_lite-{{ version }}.tar.gz
+  sha256: 5f72ba936e49eb6dafd5fb78668e50538b8ef9d1e321d937018c5cad0b4227a4
+
+build:
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - setuptools-scm >=3.4.3
+    - wheel
+    - numpy >=1.23
+    - pip
+    - gsl >=2.7
+    - swig >=4.0
+    - libblas
+    - libcblas
```

Comment:
Do you really need to specify the `blas` libraries? I think `gsl` should take of that for you.

```suggestion
```

Reply:
The build seemed to be unhappy without this:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=681128&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1854

and complained of over-linking. Maybe I missing something...

Reply:
GSL definitely needs blas, but downstream shouldn't as far as I know.

The issue with overlinking is correctly reported, `pygsl_lite` is linking unnecessarily against the gsl_cblas library. I would like to suggest the following patch:

```patch
diff --git a/recipes/pygsl_lite/meta.yaml b/recipes/pygsl_lite/meta.yaml
index a72039e8b1..2ed97fae1b 100644
--- a/recipes/pygsl_lite/meta.yaml
+++ b/recipes/pygsl_lite/meta.yaml
@@ -8,8 +8,12 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygsl_lite-{{ version }}.tar.gz
   sha256: 5f72ba936e49eb6dafd5fb78668e50538b8ef9d1e321d937018c5cad0b4227a4
+  patches:
+    - no-cblas.patch

 build:
+  ignore_run_exports:
+    - numpy
   skip: true  # [py<38 or win]
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
@@ -22,18 +26,14 @@ requirements:
     - setuptools
     - setuptools-scm >=3.4.3
     - wheel
-    - numpy >=1.23
+    - numpy
     - pip
     - gsl >=2.7
     - swig >=4.0
-    - libblas
-    - libcblas
   run:
     - python
     - numpy >=1.23
     - gsl >=2.7
-    - libblas
-    - libcblas

 test:
   imports:
```

The `no-cblas.patch` file has the following content:

```patch
diff --git a/gsl_dist/gsl_Location.py b/gsl_dist/gsl_Location.py
index 287f877..632c872 100644
--- a/gsl_dist/gsl_Location.py
+++ b/gsl_dist/gsl_Location.py
@@ -110,7 +110,7 @@ class _gsl_Location_gsl_config(_gsl_Location):

                self.prefix         = self.get_gsl_info('--prefix').strip()
                self.cflags         = self.get_gsl_info('--cflags').strip()
-               self.libs           = self.get_gsl_info('--libs').strip()
+               self.libs           = self.get_gsl_info('--libs-without-cblas').strip()
                self.config_version = self.get_gsl_info('--version')
                self.version        = self._split_version(self.config_version.strip())[:2]
```

Reply:
thanks! let me add it!

Reply:
I have changed the code upstream to not overlink